modules/gpt3module.py

```diff
@@ -48,8 +49,13 @@ def __init__(self):
         self.log_max_chars = 1500  # total log length shouldn't be longer than this
         self.log_message_max_chars = 500  # limit message length to X chars (remove the middle part)
         self.openai = OpenAI()
```

Comment:
Can you delete this extra assignment here?

Reply:
oops, now fixed